### DISCUSSION TITLE
A highlighter exquisitely for TOP

### DISCUSSION THREAD

**Discussion by Cosmic-Crafter-1** [TheOdinProject/theodinproject#3983] [Ideas]
I personally would find a highlighter tool or such very helpful since most of the extensions and tools are not able to highlight all the pages. Most major ones, recommended on chrome extension page aren't able to highlight all the pages. The others I've tried are again incapable. 
If the feature (like a specific webpage within the curriculum that contains all our highlighted texts) can be added so that we can access this to remind ourselves of some major topic we read previously, I feel it would be really helpful in the long run.
Thank you.

**Comment by CouchofTomato** [comment]
Interesting idea @Cosmic-Crafter-1 

Will definitely discuss this with the team.

**Reply by Cosmic-Crafter-1** [reply]
Thank you and appreciate all the efforts :)

**Comment by CouchofTomato** [comment]
Hey @Cosmic-Crafter-1 

I did have a small discussion on this. I think our biggest challenge is when lessons change which they sometimes do it might mess up any highlighting etc.

It might make more sense to consider a different approach where we can allow students to create some sort of notes that stick with the lesson but it would need more thought because of the structure of how lessons are created. I'll keep this open so we can update with thoughts etc.

**Reply by Cosmic-Crafter-1** [reply]
Sure, although I am not aware of the technicalities of how this operates, I've got some ideas.

If there were like a bookmark option next to each heading/sub-heading (for example : the sub-heading "growth mindset" in the motivation and mindset aspect in fundamentals section was wonderful) so if there were a small bookmark-like option just right beside the heading, clicking that would ensure that only the clicked sub-heading's contents get copied into a page called "Key Points" page INSIDE of Odin curriculum itself.
The key point's title could be the heading itself (or customizable) and the contents could then customizable as per the user's needs.
It could be beneficial since:
1. I could open this Key Points page in a new tab and if I ever needed something to back me up, I can always look into it with one click and not needing me to go all the way back (atleast me , since I'm lazy).
2. Since it will be in sync with our account, we can always open it from any device to remind ourselves just the right points at right time.
3. It will definitely save a lot of time in the long run.

I was wondering if it was worth making odin revision version system which is similar to gitHub, ( I'm still in foundations of the course, Although I do not know the exact details, so do enlighten me ), wherein if there are any major changes, we can revert to older version and still keep our notes/highlights?

Thank you.

**Reply by Cosmic-Crafter-1** [reply]
There could also be another page where we can see the most bookmarked topics with discussions below it (under 100 words to keep it simple and concise) with upvotes system or system which says why and for whom this is important (with tags like job/entrepreneurial/students in universities) that could give them a sense of more clarity. It could also along with have general mistakes made by new-comers, which can be enlightened by the professionals long enough in the field.

These are just some ideas, I would love to know why/why not these ideas are feasible so that I can think of some more. ( I love these mental creative exercises).

**Comment by CouchofTomato** [comment]
Some interesting ideas @Cosmic-Crafter-1 

Will keep them in mind for the next time we're discussing new projects for enhancing TOP.

**Reply by Cosmic-Crafter-1** [reply]
Sure, thank you.
